api/mutations/thread/publishThread.js

```diff
@@ -104,26 +104,26 @@ export default async (
 
   const { customer } = await StripeUtil.jobPreflight(community.id);
 
-  if (!customer) {
-    return new UserError(
-      'We could not verify the billing status for this channel, please try again'
-    );
-  }
-
-  const hasPaidPrivateChannel = await StripeUtil.hasSubscriptionItemOfType(
-    customer,
-    PRIVATE_CHANNEL
-  );
-  const hasFreePrivateChannel = await StripeUtil.hasSubscriptionItemOfType(
-    customer,
-    FREE_PRIVATE_CHANNEL
-  );
-
-  if (channel.isPrivate && (!hasPaidPrivateChannel && !hasFreePrivateChannel)) {
-    return new UserError(
-      'This private channel does not have an active subscription'
-    );
-  }
+  // if (!customer) {
+  //   return new UserError(
+  //     'We could not verify the billing status for this channel, please try again'
+  //   );
+  // }
+
+  // const hasPaidPrivateChannel = await StripeUtil.hasSubscriptionItemOfType(
+  //   customer,
+  //   PRIVATE_CHANNEL
+  // );
+  // const hasFreePrivateChannel = await StripeUtil.hasSubscriptionItemOfType(
+  //   customer,
+  //   FREE_PRIVATE_CHANNEL
+  // );
+
+  // if (channel.isPrivate && (!hasPaidPrivateChannel && !hasFreePrivateChannel)) {
+  //   return new UserError(
+  //     'This private channel does not have an active subscription'
+  //   );
+  // }
```

Comment:
What is all of this commented code?

Reply:
Whoops!

Reply:
Its commented because the local stripe setup breaks thread publishing if I'm running off a db backup :P